unit/deserialize_test.go

```diff
@@ -218,7 +219,7 @@ Description= words here `),
 Description= words here \
   `),
 			[]*UnitOption{
-				&UnitOption{"Unit", "Description", "words here"},
+				&UnitOption{"Unit", "Description", "words here \\"},
```

Comment:
This test shows that when you have a `\` at the end of the line followed by an empty line, that the empty line is dropped when building the `UnitOption` object.

Assuming now, that after this UnitOption are more UnitOption's, they are serialized directly after each other, right?

Initial Input
```
[Unit]
# The empty line afterwards makes this a file with 2 options
Description=long line \

Requires=docker.socket

```

```
[]UnitOptions{
  UnitOption{"Unit", "Description", "Something line with a backslash \\"},
  UnitOption{"Unit", "Required", "docker.service"},
}
```

Serializing should result into this file:

```
[Unit]
# This unitfile now has only one option
Description=long line \
Requires=docker.socket
```